### PR TITLE
wip: Freelance Services Marketplace

### PR DESCRIPTION
**Pull request by RedaDev17**
Please, commit here now ===>https://github.com/TERITORI/teritori-dapp/pull/646

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -66,3 +69,7 @@ PRICES_DB_PASSWORD=magic
 
 PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 
+#PINATA
+PINATA_API_KEY='295bb1b4e03e102493e4'
+PINATA_SECRET_API_KEY='acdecdb3a453b27ec76092dec2a455eba19da897cbff828dbac027f83b79fd74'
```
Comment (omniwired) [inline .env:74]: @0xoscario this one should be private?

File: .eslintrc.js [hunk 2]
```diff
@@ -20,4 +20,7 @@ module.exports = {
       ],
     },
   ],
+  settings: {
+    "import/ignore": ["react-native"],
```
Comment (omniwired) [inline .eslintrc.js:24]: please remove this one

File: .gitignore [hunk 3]
```diff
@@ -82,4 +82,6 @@ buck-out/
 web-build/
 dist/
 
-# @end expo-cli
\ No newline at end of file
+# @end expo-cli
+
+yarn.lock
```
Comment (omniwired) [inline .gitignore:87]: this is committed on purpose.
sometimes is needed for `vendoring` to make sure the CI pipelines can build the app as expected.

**Comment by zxxma** [comment]
Can you update PR please @0xoscario ?
Need to avoid too much conflicts, and keep this PR updated

File: .env (hunk 1 above)
Comment (WaDadidou) [reply .env]: Yes, need to use the backend way, as with Social Feed

File: .gitignore (hunk 3 above)
Comment (WaDadidou) [reply .gitignore]: Removed on https://github.com/TERITORI/teritori-dapp/pull/637

**Pull request closed without merging**